src/main/java/org/oasis_eu/portal/model/user/InvitationRequest.java

```diff
@@ -0,0 +1,33 @@
+package org.oasis_eu.portal.model.user;
```

Comment:
It's better to put in in model.authority (the old name for organization) package.